llvm/docs/AMDGPUUsage.rst

```diff
@@ -1755,6 +1755,56 @@ As part of the AMDGPU MC layer, AMDGPU provides the following target specific
 
      =================== ================= ========================================================
 
+Function Resource Usage
+-----------------------
+
+The function resource information (e.g., number of VGPRs) required depends on
```

Comment:
Nit, I would use an indefinite article to start this sentence, since it is talking about "any"/"every" function rather than a particular function. I also think consistently referring to "resource usage" rather than "resource information" would help, and the parenthetical example breaks things up too much IMO.

```
A function's resource usage depends on each of its callees' resource usage. The expressions used to denote resource usage reflect this by propagating each callees' equivalent expressions. Said expressions ...
```

Reply:
Thanks, I was having a hard time with that part so the couple of iterations of rephrasing I did might've been a bit noticeable so I do appreciate this feedback!